### PR TITLE
Some rvm fixes

### PR DESCRIPTION
**Pull request by aaronjensen**
Some of this stuff just didn't seem to be working, this makes it work

### CODE REVIEW

**Comment by aaronjensen** [comment]
@smdern please review

File: lib/cap-strap/recipes/rvm.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def self.load_into(configuration)
           end
 
           task :install_system_wide_rvm do
-            command = "curl -L https://get.rvm.io | #{sudo} bash -s stable"
+            command = "curl -L https://get.rvm.io | #{sudo} bash -s stable --without-gems=\"rubygems-bundler\""
```
Comment (smdern) [inline lib/cap-strap/recipes/rvm.rb:29]: nice

**Comment by smdern** [comment]
Looks good. nice rubygems-bunder find.

**Comment by aaronjensen** [comment]
Thanks. I noticed the specs are currently failing. I merged this in, but we should fix them.

**Pull request merged**